### DISCUSSION TITLE
Question about this Repository.

### DISCUSSION THREAD

**Discussion by nonunknown777** [louis-langholtz/PlayRho#557] [General]
Can this repo be considered a successor to Box2D library? That is still maintained? Or there are huge differences?

**Comment by louis-langholtz** [comment]
> Can this repo be considered a successor to Box2D library?

I wouldn't call PlayRho a successor to Box2D. Calling PlayRho a derived work from Box2D seems more applicable to me.

> That is still maintained?

This repo is maintained. Or do you mean Box2D? I don't know whether Box2D is being maintained.

> Or there are huge differences?

There are differences between PlayRho and Box2D. Whether they're huge, is subjective of course. Some filenames have moved or have been renamed. So just doing a filesystem compare would likely not be very telling. Taking a look at the git commit history would be more insightful in this regard.

If you can be more specific on what you mean by huge, I'll try to answer more thoroughly.

**Reply by nonunknown777** [reply]
thanks for fast response, by what I read from the repo I thought you forked box2d and keep changing/maintaining it to improve, I saw box2d repo and last update was like 10months ago, while yours seems very active, I'm learning to make a game engine, and I'm in doubt if I use yours or box2d, and would love to hear your thoughts on that.

**Reply by louis-langholtz** [reply]
> ...while yours seems very active

I'm happy to hear that PlayRho is showing up as very active!

> I'm learning to make a game engine, and I'm in doubt if I use yours or box2d, and would love to hear your thoughts on that.

Both projects have their pros and cons in my opinion. I'd be honored if you'd use PlayRho.

Incidentally, I'm hoping to release 2.0 from PlayRho's master branch soon. So if you try from the master branch, you'd more likely be using what will be released as 2.0.
